data/korath/korath missions.txt

```diff
@@ -576,3 +576,621 @@ mission "Efreti: Distress Call 8"
+			`	Spaceport authorities direct you to bay 23X-5, which contains an odd ship you see on your way in. It resembles an old-fashioned jet aircraft, whose four jet engines have been replaced with small ion engines. From the gold pipes on the tail, it appears to have a hyperdrive built into the hull. Several packages are being unloaded from the plane. A woman is waving her arms, perhaps with anger, as she walks from the cockpit into the spaceport.`
+			`	The ship is parked next to yours and the woman is about to walk past you. She looks quite distraught and, perchance, looks your way. Maybe a distraction will cheer her up?`
+			label questions
+			choice
+				`	"That's an interesting ship."`
+					goto ship
+					to display
+						not "Korath [F01]: A Fishy Deal: ship"
+				`	"What's getting you down?"`
+					goto down
+					to display
+						has "Korath [F01]: A Fishy Deal: ship"
+						not "Korath [F01]: A Fishy Deal: down"
+				`	"What sort of delivery?"`
+					goto mission
+					to display
+						has "Korath [F01]: A Fishy Deal: down"
+						not "Korath [F01]: A Fishy Deal: mission"
+				`	"Sure, I could deliver it for you."`
+					goto yup
+					to display
+						has "Korath [F01]: A Fishy Deal: mission"
+				`	"I wish I could help, but I have urgent matters of my own."`
+					decline
+					to display
+						or
+							has "Korath [F01]: A Fishy Deal: ship"
+							has "Korath [F01]: A Fishy Deal: down"
+							has "Korath [F01]: A Fishy Deal: mission"
+				`	(Ignore her and go about your business.)`
+					goto "decline mission"
+					to display
+						not "Korath [F01]: A Fishy Deal: ship"
+						not "Korath [F01]: A Fishy Deal: down"
+						not "Korath [F01]: A Fishy Deal: mission"
+			label ship
+			action
+				set "Korath [F01]: A Fishy Deal: ship"
+				log "Minor People" "Jenny Jetmaster" "Latest in a family line that has operated and maintained a replica of an ancient jet aircraft and made money on airshows. She is now on the run from the Syndicate for rescuing her persecuted uncle from Tinker."
+			`	You see her expression brighten as she walks towards you. "It's an heirloom," she says proudly, gesturing widely at the ship, "which has been in the family for centuries. My distant ancestor got a military surplus cargo plane and refurbished it. They used it as a personal craft. Each generation added to it; every piece has been repaired or replaced over the last thousand years. It's more of a replica now.`
+			`	"The whole thing has been modernized - about a dozen times. There's a hyperdrive built into the hull itself, and those ion engines aren't of Syndicate design. They're custom-built, with ion propulsion, steering, and atmospheric anti-grav, in one assembly. The design was once called a Jetmaster, and my family have taken the Jetmaster name as our own.`
```

Comment:
Here is where.

Reply:
Ah, that's easy. I'll take out "anti-grav"

> They're custom-built, with ion propulsion, steering, and atmospheric, in one assembly.

By the way, the ship is a McDonnell Douglas/Boeing C-17 Globemaster III. Being that specific in the story would cause problems. Hence, "Jetmaster."

Reply:
Sorry, but I was actually good with this reference. As previously mentioned, anti-grav as a basic principle is moderately established as being something routinely used to get spacecraft into space without needing boosters or whatnot. So this reference here is appropriate to humanity's tech level. Namely, being able to build an object that reduces the impact of gravity on the item itself and that which is attached to it. 

It's the "projecting an artificial gravity field" that I don't think is appropriate for all the possible ships the player could be flying when they encouter the series of statements.

Reply:
I did a recursive search of all txt files in the repository for the regular expression `project.*artif.*grav`, caselessly, and I don't see that anywhere. Can you tell me where it is?

The other Korath story (not in a PR yet) does have that problem, and I'll ponder a replacement, but this one doesn't use any artificial gravity fields.